.travis.yml

```diff
@@ -24,8 +24,16 @@ matrix: # Add a separate config to the matrix, using clang as compiler
     - env: ROS_DISTRO=melodic ROS_REPO=ros-shadow-fixed UPSTREAM_WORKSPACE=moveit.rosinstall
       compiler: clang
 
+# Install lxml required for create_ikfast_moveit_plugin.py
+install:
+  - sudo apt-get install -y python-lxml
```

Comment:
Good idea in general. I added it to the package.xml. 
In this particular script, we need to add it anyway, because the script can run without ROS context.